src/styles/utils/reset.scss

```diff
@@ -0,0 +1,8 @@
+body,
+p,
+h2,
+header,
+ul {
+  margin: 0;
+  padding: 0;
```

Comment:
Actually all of this elements don't have any default paddings, and header don't have both default margins and paddings (most of semantic tags, such as header, footer, main, section, article have same default properties as div)

Reply:
You forgot to fix it, next time don't styles that is not applied to tags